app/grandchallenge/cases/image_builders/tiff.py

```diff
@@ -214,52 +250,155 @@ def _add_folder_uploads(*, dzi_output, image, new_folder_upload):
     return new_folder_upload
 
 
-def image_builder_tiff(path: Path, session_id=None) -> ImageBuilderResult:
+mirax_pattern = r"INDEXFILE\s?=\s?.|FILE_\d+\s?=\s?."
+# vms (and vmu) files conatin key value pairs, where the ImageFile keys
+# can have the following format:
+# ImageFile =, ImageFile(x,y) ImageFile(z,x,y)
+vms_pattern = r"ImageFile(\(\d*,\d*(,\d)?\))?\s?=\s?.|MapFile\s?=\s?.|OptimisationFile\s?=\s?.|MacroImage\s?=\s?."
+
+
+def _get_mrxs_files(mrxs_file: Path):
+    # Find Slidedat.ini, which provides us with all the other file names
+    name, _ = os.path.splitext(mrxs_file.name)
+    slide_dat = mrxs_file.parent / name / "Slidedat.ini"
+    file_matched = []
+    if not slide_dat.exists():
+        slide_dat = [
+            f
+            for f in slide_dat.parent.iterdir()
+            if f.name.lower() == slide_dat.name.lower()
+        ][0]
+    with open(slide_dat, "r") as f:
+        lines = [
+            line for line in f.readlines() if re.match(mirax_pattern, line)
+        ]
+        for line in lines:
+            original_name = line.split("=")[1].strip()
+            file_matched.append(slide_dat.parent / original_name)
+        file_matched.append(slide_dat)
+        return file_matched
+
+
+def _get_vms_files(vms_file: Path):
+    file_matched = []
+    with open(str(vms_file.absolute()), "r") as f:
+        lines = [line for line in f.readlines() if re.match(vms_pattern, line)]
+        for line in lines:
+            original_name = line.split("=")[1].strip()
+            if os.path.exists(vms_file.parent / original_name):
+                file_matched.append(vms_file.parent / original_name)
+        return file_matched
+
+
+def _convert(
+    files: List[Path], associated_files_getter: Optional[callable], converter
+) -> List[GrandChallengeTiffFile]:
+    compiled_files: List = []
+    associated_files: List = []
+    for file in files:
+        try:
+            gc_file = GrandChallengeTiffFile(file)
+            if associated_files_getter:
+                associated_files = associated_files_getter(gc_file.path)
+            tiff_file = _convert_to_tiff(
+                path=file, pk=gc_file.pk, converter=converter
+            )
+        except Exception:
+            continue
```

Comment:
Should we really silence all errors here? We should be explicit about the errors we expect and silencing those is ok, but 
I think that we should have some way of knowing when something unexpected is happening.